packages/adminrouter/extra/src/README.md

```diff
@@ -24,9 +24,26 @@ Admin Router runs on both master and agent nodes, each with different configurat
 - HMTL: [docs/api/nginx.agent.html](docs/api/nginx.agent.html)
 - Rendered: <https://rawgit.com/dcos/dcos/master/packages/adminrouter/extra/src/docs/api/nginx.agent.html>
 
-Use `make api-docs` to regenerate the YAML and HTML files.
 
-Use `make check-api-docs` to validate that the YAML and HTML files are up to date.
+## Endpoints documentation
+
+All Admin Router endpoints are documented using the
+[ngindox](https://github.com/karlkfi/ngindox) tool, which uses specially
+formatted comments in order to describe endpoint configurations, and
+automatically parses them into HTML documents. Please check the project's
+documentation for more details.
+
+Admin Router's CI automatically checks if the endpoint documentation generated
+using `ngindox` and embedded into the repository is up to date. If not, the CI
+job fails and the user needs to regenerate the docs and re-submit the PR.
+
+The check is done by generating the documentation during the build stage.
+If, after the `ngindox` run, `git` detects uncommitted changes, then this means
+that the Admin Router configuration differs from the HTML documents that
+are committed into repository. This is done using `make check-api-docs` target.
+
+In order to regenerate the documentation files, one needs to execute the
```

Comment:
That link isn't for "committed into repository" as per the text. In fact Ludwig doesn't have relevant results for that - https://ludwig.guru/s/committed+into+repository.